pairing_bot.go

```diff
@@ -193,6 +214,18 @@ func (pl *PairingLogic) match(w http.ResponseWriter, r *http.Request) {
 	if err := pl.pdb.SetNumPairings(ctx, int(timestamp), numPairings); err != nil {
 		log.Printf("Failed to record today's pairings: %s", err)
 	}
+
+	for _, r := range inactive {
+		if err := pl.rdb.Delete(ctx, r.id); err != nil {
```

Comment:
Maybe: send the message, and then only delete if we successfully sent the message?

I'd hate to see someone "silently" unsubscribed if there's a bug here.